csharp/src/Google.Protobuf.Test.CodeGen/Google.Protobuf.Test.CodeGen.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netstandard1.0</TargetFrameworks>

```

Comment:
I think we'd still want to include netstandard2.0 (but we can keep netstandard1.0)?

Can you also add a comment into the .csproj file to explain why the project with generated code is separate? There are several good reasons.